frontend/public/components/factory/table.tsx

```diff
@@ -264,13 +267,27 @@ export type TableRowProps = {
   className?: string;
 };
 
-export const TableData: React.SFC<TableDataProps> = ({ className, ...props }) => {
-  return <td {...props} className={className} role="gridcell" />;
+export const TableData: React.SFC<TableDataProps> = ({
+  className,
+  columnID,
+  columns,
+  ...props
+}) => {
+  const showNamespace =
+    columnID !== 'namespace' || UIActions.getActiveNamespace() === ALL_NAMESPACES_KEY;
+  return (
+    ((!_.isEmpty(columns) ? columns.has(columnID) : true) && showNamespace && (
+      <td {...props} className={className} role="gridcell" />
+    )) ||
+    null
+  );
```

Comment:
```suggestion
  return (
    (_.isEmpty(columns) || columns.has(columnID) && showNamespace
      ? <td {...props} className={className} role="gridcell" />
      : null
  );
```

Reply:
This doesn't properly filter out the namespaces for non col management tables...keeping orig return

Reply:
> This doesn't properly filter out the namespaces for non col management tables...keeping orig return

I think I just missed a closing paren. Does this work?

```suggestion
  return (
    (_.isEmpty(columns) || columns.has(columnID)) && showNamespace
      ? <td {...props} className={className} role="gridcell" />
      : null
  );
```